src/main/java/seedu/address/model/person/Contact.java

```diff
@@ -53,7 +53,9 @@ public Address getAddress() {
         return address;
     }
 
-    public Type getType() { return type; }
+    public Type getType() {
```

Comment:
you should not have a code change if it's just syntax, for purposes of git-blame